packages/components/e2e/component-objects/src/test/java/org/talend/config/LocalWebDriverTest.java

```diff
@@ -0,0 +1,16 @@
+package org.talend.config;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class LocalWebDriverTest extends WebDriverTest {
+
+    public void setUp() {
+        System.out.println("Using WebDriver from " + System.getProperty("webdriver.chrome.driver"));
```

Comment:
Why didn't you used the `Logger` used in other classes ?

Reply:
👌